src/agent/configuration_parser/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ find_package(yaml-cpp CONFIG REQUIRED)
 
 add_library(ConfigurationParser src/configuration_parser.cpp)
 target_include_directories(ConfigurationParser PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
-target_link_libraries(ConfigurationParser PUBLIC yaml-cpp::yaml-cpp Logger)
+target_link_libraries(ConfigurationParser PUBLIC yaml-cpp::yaml-cpp Logger Config)
```

Comment:
Looks like `Config` can be `PRIVATE` here.

Reply:
Done!